languages/purescript/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+PureScript is a statically-typed language, meaning that the type of a variable is checked when the program is compiled. Using the `=` operator, we can bind a value to a variable name:
+
+```purescript
+variable = 10 -- Bind the integer value 10
```

Comment:
I think the comments section should be above this, because if I don't know that this is a comment, I might think that the `-- Bind the integer value 10` is part of "defining a variable".